src/App.svelte

```diff
@@ -265,21 +288,17 @@
           }
         }
 
-        new_videos[video.UAR] = video;
+        media_store.push(video, false);
```

Comment:
Won't this trigger an update on gdocs? Maybe here we can directly set the $media_store with an assignment.

Reply:
nope, see pushStore code, (and commit message) we do a deep compare there before updating the doc.
google-spreadsheet also does that btw.

Reply:
see [af0a13c](https://github.com/edipoarg/codec/pull/2/commits/af0a13c73d7b08ee339eab41c976d8c18dcc12be)

Reply:
But on the first assignment media_store is an empty object, right?

Reply:
nope because it comes with emit = false on that code path.

Reply:
:see_no_evil: